aptos-move/framework/aptos-framework/sources/stake.move

```diff
@@ -427,11 +448,15 @@ module aptos_framework::stake {
     }
 
     /// Allow on chain governance to remove validators from the validator set.
+    ///
+    /// This is a system-level validator set change, therefore not affected by user-level locking.
+    /// Caller should ensure a fast reconfiguration is triggered after this.
     public fun remove_validators(
         aptos_framework: &signer,
         validators: &vector<address>,
-    ) acquires ValidatorSet {
+    ) acquires ValidatorSet, ValidatorSetChangeLockStatus {
```

Comment:
@zekun000 @movekevin will this have backward-compatibility issue?

Reply:
I "think" this is fine

Reply:
Nope changing acquires list is not subject to compat checks. Only the function's type arg, params and return types  are